pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -328,8 +328,10 @@ let
         # Note that not all libraries are prefixed with "lib",
         # eg. glibc-2.30/lib/ld-2.30.so
         "mr $path/lib/**.so*"
+        "mr $path/lib64/**.so*"
```

Comment:
So in nixpkgs lib64 is usually a symlink, but apparmor might also need symlink location in its whitelist. At least the current behavior suggests that.

Reply:
AppArmor usually resolves symlinks when finding a file